plaidml/bridge/openvino/tests/functional/plugin/plaidml/shared_tests_instances/single_layer_tests/mod.cpp

```diff
@@ -0,0 +1,27 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <cstddef>
+#include <map>
+#include <vector>
+
+#include "common_test_utils/test_constants.hpp"
+#include "single_layer_tests/mod.hpp"
+
+using LayerTestsDefinitions::ModLayerTest;
+
+namespace {
+const std::vector<InferenceEngine::Precision> netPrecisions = {
+    InferenceEngine::Precision::FP32,
+};
+
+const std::vector<std::vector<std::size_t>> inputShapes = {
+    {std::vector<std::size_t>({1, 30}), std::vector<std::size_t>({1, 30})}};
```

Comment:
Can you make the inputs be different-but-broadcastable shapes? e.g. (5, 10, 3) and (1, 10, 1). If I understand how the test infrastructure generates inputs correctly, what you currently have will just test `1 % 1`, `2 % 2`, `3 % 3`, etc, which isn't a great test of mod.

Reply:
Added this as a new test (and made some styling improvements)